tests/multipleBackend/multipartUpload.js

```diff
@@ -0,0 +1,253 @@
+const assert = require('assert');
+const AWS = require('aws-sdk');
```

Comment:
do we have any tests where we overwrite a key that was put to file by completing an mpu with the same key name and the other way around to make sure we are cleaning up correctly?